### ISSUE TITLE
Mac OS: playlists not exposed to VoiceOver and missing button labels

### ISSUE THREAD

**Issue by pitermach** [Alexey-Yakovenko/deadbeef#1650]
STR:

- Open deadbeef
- Press Command-F5 to enable VoiceOver
- You can use ctrl-option and the arrow keys to navigate the interface, left/right for next and previous control, down/up to enter and leave containers like scroll areas.

There are 2 major accessibility problems with deadbeef's Cocoa UI for VO. The first is that buttons and sliders have no label or just say Custom view, but more importantly it doesn't read the playlist table, the best I can get is that it picks up an "empty scroll area". With my somewhat limited understanding of the accessibility protocol I think this shouldn't be too hard to fix and is mostly a matter of telling VO what should be the table and the rows.

**Comment by ekavun** [comment]
Hello hello,
  Blind person here, and I can confirm this is still an issue testing on DeaDBeef Version 1.0 (ad43804). I'm on Mac OS 10.15.7.
I was really excited to finally find a real music player. Seems really efficient in ingesting large folders and stuff, but I cannot navigate at all the list of tracks. I basically can't do much besides using the preferences dialog, and use the built-in keyboard shortcuts. The main area seems to be rendered as a Scroll area that appears as "empty" to VoiceOver.
So, no notion of a cursor / focus, can't read the list of tracks or playlists, etc.

The software sounds amazing, and if it's not too much trouble, would love to see some "accessibility" action. I'm pretty convinced by your piece as it is, so if this turns into a bounty, let me know. :)

All the best, and happy 2021, while we still can!
